### PR TITLE
fixes for DEVTOOLBPS-23,25,26

### PR DESCRIPTION
**Pull request by Drifftr**

### CODE REVIEW

File: plugins/org.wso2.developerstudio.humantaskeditor/HumanTaskEditor/js/WSDLOperations.js [hunk 1]
```diff
@@ -56,25 +56,27 @@ function generateHTConfig(configDom, xmlDom, currentTask) {
     wsdlHTDeploymentConfig = configDom.getElementsByTagName("HTDeploymentConfig")[0];
 
     tasksList = xmlDom.getElementsByTagName("task");
+    targetnamespace = xmlDom.getElementsByTagName("humanInteractions")[0].getAttribute("targetNamespace");
```
Comment (himasha) [inline plugins/org.wso2.developerstudio.humantaskeditor/HumanTaskEditor/js/WSDLOperations.js]: Add the tag names as constants.

Comment (Drifftr) [inline plugins/org.wso2.developerstudio.humantaskeditor/HumanTaskEditor/js/WSDLOperations.js]: Fixed in https://github.com/wso2/devstudio-tooling-bps/pull/25

File: plugins/org.wso2.developerstudio.humantaskeditor/HumanTaskEditor/js/WSDLOperations.js [hunk 2]
```diff
@@ -101,10 +103,13 @@ function generateInputWSDL(wsdlDom, currentTask) {
     serviceName = currentTask+"Service";
     portName = currentTask + "Port";
     inputElements = $('#' + currentTaskName + ' #inputmappingTable tr');
-
+    targetnamespace = xmlDom.getElementsByTagName("humanInteractions")[0].getAttribute("targetNamespace");
+    wsdlDom.getElementsByTagName("definitions")[0].setAttribute("xmlns:tns",targetnamespace);
+    wsdlDom.getElementsByTagName("definitions")[0].setAttribute("targetNamespace",targetnamespace);
```
Comment (nirmal070125) [inline plugins/org.wso2.developerstudio.humantaskeditor/HumanTaskEditor/js/WSDLOperations.js]: NPE is possible in above lines if the 0th element is null.

Comment (Drifftr) [inline plugins/org.wso2.developerstudio.humantaskeditor/HumanTaskEditor/js/WSDLOperations.js]: Thanks. Null checks were added in https://github.com/wso2/devstudio-tooling-bps/pull/30/commits/acbad877337a7e63dd99f935d9d2c3329ab6c1de

Comment (Drifftr) [inline plugins/org.wso2.developerstudio.humantaskeditor/HumanTaskEditor/js/WSDLOperations.js]: Thanks. Array length checks were added in acbad87

Comment (Drifftr) [inline plugins/org.wso2.developerstudio.humantaskeditor/HumanTaskEditor/js/WSDLOperations.js]: Thanks. Array length checks were added in https://github.com/wso2/devstudio-tooling-bps/pull/30/

**Pull request merged**